train.py

```diff
@@ -270,8 +268,7 @@ def write_bag_cache(bag_file: str, bag_cache_path: str, backbone_onnx_path: str,
     parser.add_argument('--epoch-steps', type=int, default=100, help='how often to save checkpoints')
     parser.add_argument('--seed', type=int, default=None, help='training seed')
     parser.add_argument('--lstm-history', type=int, default=240, help='max amount of prior steps to feed into a network history')
-    parser.add_argument('--history-indexes', type=str, default='-1,-2,-3,-5,-8,-13,-21,-34,-55,-89,-144,-233',
-                        help='which indexes to pass into the network')
+    parser.add_argument('--history-indexes', type=str, default='-1,-2,-3,-5', help='which indexes to pass into the network')
```

Comment:
Leftover from testing?

Reply:
No, decided to just go down to 1,2,3,5 on purpose, not the huge history of the fibonacci tests.